src/main/java/no/finn/unleash/strategy/StrategyUtils.java

```diff
@@ -0,0 +1,55 @@
+package no.finn.unleash.strategy;
+
+public final class StrategyUtils {
+    private static final int ONE_HUNDRED = 100;
+
+    public static boolean isNotEmpty(final CharSequence cs) {
+        return !isEmpty(cs);
+    }
+
+    public static boolean isEmpty(final CharSequence cs) {
+        return cs == null || cs.length() == 0;
+    }
+
+    public static boolean isNumeric(final CharSequence cs) {
+        if (isEmpty(cs)) {
+            return false;
+        }
+        final int sz = cs.length();
+        for (int i = 0; i < sz; i++) {
+            if (Character.isDigit(cs.charAt(i)) == false) {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    /**
+     * Takes to string inputs concat them, produce a hashCode and return a normalized value between 0 and 100;
+     *
+     * @param identifier
+     * @param groupId
+     * @return
+     */
+    public static int getNormalizedNumber(String identifier, String groupId) {
+        int hashCode = Math.abs((groupId + ':' + identifier).hashCode());
+        return hashCode % ONE_HUNDRED + 1;
+    }
+
+    /**
+     * Takes a numeric string value and converts it to a integer between 0 and 100.
+     *
+     * returns 0 if the string is not numeric.
+     *
+     * @param percentage - A numeric string value
+     * @return a integer between 0 and 100
+     */
+    public static int getPercentage(String percentage) {
+        if (isNotEmpty(percentage) && isNumeric(percentage)) {
```

Comment:
Not to important after the UI got input types. Percentage input looks like this:
![image](https://cloud.githubusercontent.com/assets/158948/22640305/52a85f38-ec52-11e6-9df7-8345c0dc90a6.png)